client/i18n-unused.config.js

```diff
@@ -3,6 +3,38 @@
 module.exports = {
   localesPath: 'packages/common/src/intl/locales',
   srcPath: 'packages',
+  srcExtensions: ['ts', 'tsx'],
+  excludeKey: [
+    'approval-status.',
+    'button.back',
+    'button.ok-and-next',
+    'description.',
+    'error.',
+    'gender.',
+    'inventory-addition',
+    'inventory-reduction',
+    'label.',
```

Comment:
why excluding all labels? and descriptions?

Reply:
this will be resolved with new table implementation, so might be easier to review then!

happy to merge this whenever, we can do the review of outstanding keys separately 🙏